core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -2804,6 +2804,17 @@ payment.f2f.info.openURL=Open web page
 payment.f2f.offerbook.tooltip.countryAndCity=Country and city: {0} / {1}
 payment.f2f.offerbook.tooltip.extra=Additional information: {0}
 
+# Japan Bank Transfer
+payment.japan.bank=Bank or Financial Institution
+payment.japan.bank.select=Search for Financial Institution
+payment.japan.bank.code=Bank Code
+payment.japan.bank.name=Bank Name
+payment.japan.bank.branch.code=Branch Code
+payment.japan.bank.branch.name=Branch Name
+payment.japan.bank.account.number=Account Number
+payment.japan.bank.account.name=Account Name
+payment.japan.bank.account.type=Account Type
```

Comment:
To prevent unnecessary translations you could use `payment.accountType` instead.